docs/04-upload.md

```diff
@@ -126,7 +126,7 @@ First, you need to provide a JSON file containing the conversation messages foll
 [
   {
     "who": "ORIGIN_UUID",
-    "to": ["DESTINATION_UUID"],
+    "destination": ["DESTINATION_UUID"],
```

Comment:
Is this an intented change? as far as I know the conversation field payload didn't change